Week-12/todo-list/src/Components/Todo.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import '../Style/Todo.css';
+import ListItem from "./ListItem";
+
+const Todo = () => {
+
+    const [list, setList] = useState(['Coding', 'Eating', 'Sleeping', 'Debugging']);
+    const addItem = () => {
+        setList([...list, '']);
```

Comment:
this will add '' empty string in the array. This won't work I guess

Reply:
Yes, it is for adding new entry in the todo list. But I have changed the implementation as per the comment